src/compile/scale/properties.ts

```diff
@@ -102,6 +102,7 @@ export function parseNonUnitScaleProperty(model: Model, property: keyof (Scale |
         // TODO: precedence rule
         valueWithExplicit = mergeValuesWithExplicit(
           valueWithExplicit, childValueWithExplicit,
+          property,
```

Comment:
Well, you need to know what's the property to make the warning message include the property name.